README.rst

```diff
@@ -10,48 +10,95 @@ pytest-picked
     :target: https://ci.appveyor.com/project/anapaulagomes/pytest-picked/branch/master
     :alt: See Build Status on AppVeyor
 
-Run the tests related to the changed files (according with Git)
+.. image:: https://img.shields.io/pypi/v/pytest-picked.svg
+    :target: https://pypi.org/project/pytest-picked/
+    :alt: See Package Status on PyPI
 
-----
+Run the tests related to the modified files (according to Git)
 
-This `pytest`_ plugin was generated with `Cookiecutter`_ along with `@hackebrot`_'s `cookiecutter-pytest-plugin`_ template.
+---
 
+.. image:: demo.gif
+    :target: https://asciinema.org/a/cWV3sx6poG4aaPZfyVI8nOJ7J
+    :height: 400px
+    :alt: Demo
 
-Features
---------
+Let's say you have the following output from your ``git status``:
 
-* Run the tests according with changed files from `git status`
+::
 
+  $ git status
 
-Installation
-------------
+  On branch master
+  Your branch is ahead of 'origin/master' by 1 commit.
+    (use "git push" to publish your local commits)
 
-You can install "pytest-picked" via `pip`_ from `PyPI`_::
+  Untracked files:
+    (use "git add <file>..." to include in what will be committed)
 
-    $ pip install pytest-picked
+    api.py
+    tests/api/
+    tests/test_board.py
+
+  nothing added to commit but untracked files present (use "git add" to track)
+
+
+Running ``pytest --picked``, the plugin will run all the tests that comes from this output.
+
+::
+
+  $ pytest --picked
+
+  ============================= test session starts =============================
+  platform darwin -- Python 3.6.4, pytest-3.6.0, py-1.5.3, pluggy-0.6.0
+  rootdir: /Users/ana.gomes/personal-workspace/grandma, inifile:
+  plugins: picked-0.1.0, mock-1.10.0, flask-0.10.0, deadfixtures-2.0.1
+  collecting 34 items
+  Changed test files... 1. ['tests/test_board.py']
+  Changed test folders... 1. ['tests/api/']
+  collected 34 items
+
+  tests/test_board.py .                                                      [ 50%]
+  tests/api/test_new.py .                                                    [100%]
+
+  =========================== 2 passed in 0.07 seconds ===========================
+
+The files and folders with tests, that were modified but weren't committed yet will be run.
+No more copy and paste!
 
 
 Usage
 -----
 
-`pytest --picked`
+::
 
+  $ pytest --picked
 
-TODO
------
 
-* Add option to run the changed files from the current branch
-* Guess the test file from given changed file
```

Comment:
Ah, great to remove these. I was confused when I first read these. Glad you moved them into issues.